staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -542,10 +557,23 @@ func kmsPrefixTransformer(config *apiserverconfig.KMSConfiguration, stopCh <-cha
 			l:            &sync.Mutex{},
 			lastResponse: &kmsPluginHealthzResponse{},
 		}
+		// initialize keyID
+		keyID := ""
+		probe.keyID.Store(&keyID)
+
+		go wait.PollImmediateUntilWithContext(
```

Comment:
Trying to think if there is a way to have an integration test that proves that this loop is running. 🤔